lib/reline/line_editor.rb

```diff
@@ -94,7 +94,7 @@ def simplified_rendering?
     mode_string
   end
 
-  private def check_multiline_prompt(buffer)
+  private def check_multiline_prompt(buffer, force_recalc = false)
```

Comment:
Can we make `force_recalc` a keyword argument? It'll help understanding the callee's action without going back here.